crates/cli/tests/runner/mod.rs

```diff
@@ -89,11 +99,17 @@ impl Runner {
         run.call(&mut store, ())?;
         let store_context = store.into_data();
         drop(store_context.wasi);
-        Ok(store_context
+        let logs = store_context
+            .log_stream
+            .try_into_inner()
+            .expect("Output stream reference still exists")
```

Comment:
Should probably update this text to indicate it's referring to the log stream or stderr and not the output